### ISSUE TITLE
Need maintainer access to put back master in a correct state 😇 

### ISSUE THREAD

**Issue by johnraz** [jazzband/django-rest-knox#341]
I made a mistake today while trying to get some commits that were wrongly put on master on develop (see: https://github.com/jazzband/django-rest-knox/pull/337)

Seems like while resolving conflict it merged develop on master ... 

I was abe to revert the change https://github.com/jazzband/django-rest-knox/pull/339 but it made the matter worse as now master things it's ahead of develop and trying to merge develop into master has no effect anymore 🤦 

TIL don't trust integrated editor in GH 😇 

The easiest way out would be to force push master to the original commit before any merge from develop happened the commit is: `5c128c7787ba703d29a1b28bbe2c45973208fec6` (https://github.com/jazzband/django-rest-knox/commit/5c128c7787ba703d29a1b28bbe2c45973208fec6) 

Basically doing:
```
git checkout master
git reset --hard 5c128c7787ba703d29a1b28bbe2c45973208fec6
git push -f origin master
``` 

But in order to do that we need to allow force push on master for a little while and I don't have the permissions to do so :/

@giovannicimolin do you have maintainer rights?

I will also try through the jazzband help repo

I also added a message to the readme on master to try to raise awareness of this situation (https://github.com/jazzband/django-rest-knox/commit/837b9e4dea09d9c4568864e0c05f82fe0d61a364)

Very sorry for the noise 😢
